hazelcast/src/main/java/com/hazelcast/nio/tcp/handlermigration/IOBalancer.java

```diff
@@ -178,26 +179,4 @@ private void tryMigrate(LoadImbalance loadImbalance) {
         }
         handler.migrate(destinationSelector);
     }
-
-    private final class IOBalancerThread extends Thread {
```

Comment:
the original idea was to improve testability. I didn't help great deal, but I like it outside anyway.